linker/Linker.Steps/MarkStep.cs

```diff
@@ -453,6 +453,10 @@ protected virtual bool ShouldMarkCustomAttribute (CustomAttribute ca, ICustomAtt
 				case "System.ThreadStaticAttribute":
 				case "System.ContextStaticAttribute":
 					return true;
+				// Attributes related to `fixed` keyword used to declare fixed length arrays
+				case "System.Runtime.CompilerServices.FixedBufferAttribute":
```

Comment:
Only FixedBufferAttribute is needed to be persisted

Reply:
Fixed

Reply:
Actually, on second look. I think we should move it out of _context.KeepUsedAttributeTypesOnly as well.

Reply:
Why is that?  We keep all attributes when `!_context.KeepUsedAttributeTypesOnly`

Reply:
fyi, when I made the first fix I added a version of the test for when  `_context.KeepUsedAttributeTypesOnly` is not enabled.  That will give us test coverage on the behavior when `!_context.KeepUsedAttributeTypesOnly`